book.json

```diff
@@ -9,6 +9,8 @@
     "anchors"
   ],
   "variables": {
-    "org": "Root Systems"
+    "org": "Root Systems",
+    "title": "Enspiral Root Systems NZ",
```

Comment:
i thought `title` and `description` were top-level properties in the `book.json`: https://toolchain.gitbook.com/config.html. do these values show up in the `<title></title>` and `<meta name="description></meta>` tags?